extension/src/popup/helpers/parseTransaction.ts

```diff
@@ -5,18 +5,35 @@ import { MEMO_TYPES } from "popup/constants/memoTypes";
 
 import { ErrorMessage } from "@shared/api/types";
 
-export const decodeMemo = (memo: {}) => {
-  const memoType = get(memo, "_switch.name", "");
+type Memo = {
```

Comment:
Should we use the type from stellar base here?
https://github.com/stellar/js-stellar-base/blob/cf370f4e627e2b0e296ceceaebd0164dede396bd/types/index.d.ts#L212